.dependabot/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 1
+# Config Docs: https://dependabot.com/docs/config-file/#available-configuration-options
+
+update_configs:
+  # Javascript package.json updates
+  - directory: "/server"
+    package_manager: "javascript"
+    update_schedule: "daily"
```

Comment:
given our review schedule, maybe we want "weekly", so that these upgrades happen slowly for now?

Reply:
daily is too much, IMO. weekly or less seems more appropriate.

Reply:
```suggestion
    update_schedule: "weekly"
```